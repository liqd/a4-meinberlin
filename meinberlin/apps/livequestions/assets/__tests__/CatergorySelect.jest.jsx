import React from 'react'
import { render } from '@testing-library/react'
import SelectCategory from '../CategorySelect'

test('SelectCategory renders correctly', () => {
  const { asFragment } = render(
    <SelectCategory
      name="Category"
      category_dict={{ 123: 'category1', 124: 'category2' }}
    />)
  expect(asFragment()).toMatchSnapshot()
})
