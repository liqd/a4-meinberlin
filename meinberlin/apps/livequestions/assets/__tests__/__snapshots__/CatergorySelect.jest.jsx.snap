// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectCategory renders correctly 1`] = `
<DocumentFragment>
  <div>
    <div
      class="live_questions__select u-spacer-bottom"
    >
      <label
        for="categorySelect"
      >
        Affiliation*
      </label>
      <div
        class="form-hint"
      >
        Answered questions will be displayed in the statistics according to the chosen affiliation.
      </div>
      <select
        class="js-select2"
        data-minimum-results-for-search="Infinity"
        id="categorySelect"
        name="categorySelect"
        required=""
      >
        <option
          value=""
        >
          --------
        </option>
        <option
          value="123"
        >
          category1
        </option>
        <option
          value="124"
        >
          category2
        </option>
      </select>
    </div>
  </div>
</DocumentFragment>
`;
